src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -360,7 +360,7 @@ paths:
             type: string
```

Comment:
Rotation could definitely use a default of 0 and some examples where at least one is not 0, 90, 180 or 270.

Reply:
Examples now added